handlers/message.go

```diff
@@ -0,0 +1,104 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/alecthomas/kong"
+	"github.com/bcneng/candebot/bot"
+	"github.com/bcneng/candebot/cmd"
+	"github.com/bcneng/candebot/inclusion"
+	"github.com/bcneng/candebot/slackx"
+	"github.com/newrelic/newrelic-telemetry-sdk-go/telemetry"
+	"github.com/slack-go/slack/slackevents"
+	"log"
+	"strings"
+	"time"
+)
+
+func MessageEventHandler(botCtx bot.Context, e slackevents.EventsAPIInnerEvent) error {
+	event := e.Data.(*slackevents.MessageEvent)
```

Comment:
nit: should we be defensive there and check for `ok`?

Reply:
If we really want to super perfect, yes. But I skipped that check in purpose since this is kinda done in the `CreateEventHandler` method.